### DISCUSSION TITLE
How to draw a line with background

### DISCUSSION THREAD

**Discussion by crazyfireji** [JuliaPlots/Makie.jl#4264] [Q&A]
I want to plot a line on the mesh, but the line will covered by mesh(as the fig following), is there any method to plot a line with background???
![image](https://github.com/MakieOrg/Makie.jl/assets/112403698/7926d25e-7fc2-4038-8216-4b6f9d8e13fa)

**Comment by asinghvi17** [comment]
`translate!(lineplot, 0, 0, 10)` should move the line in front of the mesh.  You can either do that, plot with `Point3f` with explicit z value, or plot the line after the mesh (CairoMakie only!).
